File: lib_dirty/src/main/java/com/ghl/lib_dirty/base/BaseActivity.java
package com.ghl.lib_dirty.base;

import android.content.Context;
import android.os.Bundle;
import android.view.Window;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public abstract class BaseActivity extends AppCompatActivity {

    protected Context mActivity;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        setBaseConfig();
        super.onCreate(savedInstanceState);
        mActivity = this;

    }
    private void setBaseConfig() {
        requestWindowFeature(Window.FEATURE_NO_TITLE);
    }


    @Override
    protected void onDestroy() {
        super.onDestroy();
    }
}

File: lib_common/src/main/java/com/ghl/common/base/BaseActivity.kt
package com.ghl.common.base

import android.content.Context
import android.os.Bundle
import android.view.Window
import androidx.activity.ComponentActivity
import androidx.appcompat.app.AppCompatActivity

abstract class BaseActivity : ComponentActivity() {
    protected var mActivity: Context? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        setBaseConfig()
        super.onCreate(savedInstanceState)
        mActivity = this
    }

    private fun setBaseConfig() {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
    }

}